pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -36,7 +36,7 @@
 public class PinotHelixTaskResourceManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(PinotHelixTaskResourceManager.class);
 
-  public static final String TASK_NAME_SEPARATOR = "_";
+  public static final String TASK_NAME_SEPARATOR = "__";
```

Comment:
Can you make sure that this class also filters task queue names to make sure we can't create a task queue called something like `taskQueue_OFFLINE`, which would be table name.